core/renderer.go

```diff
@@ -35,13 +40,17 @@ func (r *Renderer) NewCursor() *terminal.Cursor {
 	}
 }
 
-func (r *Renderer) Error(invalid error) error {
+func (r *Renderer) Error(invalid error, icon string) error {
```

Comment:
For consistency, I think we should pass in the IconSet here.  Then the template would look like `{{.Icons.Error}}`. I realize that only one Icon is used for now, but perhaps we will add another icon eventually or a user will want to make a more elaborate ErrorTemplate that utilizes other icons?

Reply:
Yea I had that initially but then moved the `IconSet` definition to `survey` so that `WithIconSet`s signature looked reasonable which created a circular import since we render the Error template in `core`. Maybe the solution is to move `Renderer` into the root package?